ts/features/bonus/common/components/BonusInformationComponent.tsx

```diff
@@ -108,6 +108,7 @@ const BonusInformationComponent: React.FunctionComponent<Props> = props => {
     title: I18n.t("global.buttons.cancel")
   };
   const requestButtonProps = {
+    buttonFontSize: customVariables.btnFontSize - 1,
```

Comment:
For accessibility reasons I guess decrease the font-size is not a good idea.
Perhaps we can change the text and make it shorter

see https://www.pivotaltracker.com/story/show/175090703/comments/218618829

Reply:
Changed the label as suggested.

<img src="https://user-images.githubusercontent.com/11773070/95436856-a2503380-0954-11eb-8c49-211ac4007526.png" width=300> <img src="https://user-images.githubusercontent.com/11773070/95436892-a9774180-0954-11eb-9154-d9e6729d9656.png" width=300>